Makefile

```
@@ -66,16 +68,16 @@ build-ui: install-ui ## Build ui
 
 build: build-ui ## Build bifrost-http binary
 	@echo "$(GREEN)Building bifrost-http...$(NC)"
-	@cd transports/bifrost-http && go build -o ../../tmp/bifrost-http .
+	@cd transports/bifrost-http && GOWORK=off go build -o ../../tmp/bifrost-http .
 	@echo "$(GREEN)Built: tmp/bifrost-http$(NC)"
 
 run: build ## Build and run bifrost-http (no hot reload)
```

checkmake (0.2.2)

[warning] Target body for "run" exceeds allowed length of 5 (7).

(maxbodylength)